### PR TITLE
Added documentation for voters

### PR DESCRIPTION
**Pull request by Nek-**

### CODE REVIEW

File: doc/05-Voters.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+Vote for the current page
+=========================
+
+Basics
+------
+
+To guess what's the current page inside your menu, KnpMenu use a Voter mechanism.
+
+You have to define an `ItemMatcher` that use one or many voter to decide if the page is or not the current page.
```
Comment (stof) [inline doc/05-Voters.md]: you don't define a matcher. You instantiate it. The class definition is already part of KnpMenu

File: doc/05-Voters.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+class MyAwesomeVoter implements VoterInterface
+{
+    // Your code
+}
+```
+
+> You also can create your own matcher if you want to add more logic into your menus.
+
+
+*Note: Voters are natively implemented inside KnpMenuBundle.*
```
Comment (stof) [inline doc/05-Voters.md]: it looks weird to me to talk about KnpMenuBundle here. The library should not be aware of the bundle IMO.

Comment (dbu) [inline doc/05-Voters.md]: what exactly did you try to say here @Nek- ? something like

"When using KnpMenuBundle in Symfony, the RouterVoter is activated by default." ??

as the bundle will not repeat the explanation of the concept of the voters (i guess) , this makes sense to add a note about this.

Comment (Nek-) [inline doc/05-Voters.md]: It's to avoid questions like « I'm trying to use voter with symfony » in issues. And yes your sentence is better @dbu  :-) .

Comment (dbu) [inline doc/05-Voters.md]: is it the RouterVoter or all voters?

**Comment by stof** [comment]
the builtin voters should also be documented, to show their features (the RouteVoter can vote based on several routes for insrance)

File: doc/05-Voters.md (hunk 2 above)
Comment (stof) [inline doc/05-Voters.md]: the KnpMenuBundle documentation needs to talk about the way to register voters (it relies on a tag). This is the place where the doc should mention that the RouterVoter is already registered automatically.

**Comment by stof** [comment]
@Nek- please finish the work on this PR

**Comment by dbu** [comment]
ping @Nek-

**Comment by dbu** [comment]
continued in #246

**Pull request closed without merging**